src/universal/utils/dotenv.js

```diff
@@ -9,12 +9,15 @@ import dotenvExpand from 'dotenv-expand';
  * Returns true.
  */
 export default function getDotenv() {
-  const myConfig = {silent: true};
-  if (process.env.NODE_ENV === 'test') {
-    myConfig.path = '.env.test';
+  const {NODE_ENV} = process.env;
+
+  const config = {silent: true};
+
+  if (NODE_ENV) {
+    config.path = `.env.${NODE_ENV}`;
```

Comment:
I think this change will have implications for others' dev environments, so I can revert it if need be. The main point is that this used to only use `.env.test` when `NODE_ENV === "test"`, and I changed it to use `.env.${whateverTheNodeEnvIs}`. I'm finding it useful to keep a bunch of local `.env` files for different builds.

Reply:
i'm not a huge fan of the multiple `.env` files. it may be a necessary evil for running a test on CI, but i'd be more inclined to get rid of the `.env.test` and move that stuff to the CI internal environment instead of expanding the scope. see https://github.com/motdotla/dotenv#should-i-have-multiple-env-files

Reply:
I think this is fine, as long as `.env*` is in `.gitignore`. It's a fine thing to be able to be able to run your local instance and fire up the e2e tests for local integration (or even local continuous integration). I'd also suggest a fallback to `.env` if it cannot find the local `.env.$NODE_ENV` file...